lib/python/src/bailo/helper/model.py

```diff
@@ -111,14 +111,44 @@ def from_id(cls, client: Client, model_id: str) -> Model:
 
         return model
 
+    @classmethod
+    def search(
+        cls,
+        client: Client,
+        task: str | None = None,
+        libraries: list[str] | None = None,
+        filters: list[str] | None = None,
+        search: str = "",
+    ) -> list[Model]:
+        """Return a list of model objects from Bailo, based on search parameters.
+
+        :param client: A client object used to interact with Bailo
+        :param task: Model task (e.g. image classification), defaults to None
+        :param libraries: Model library (e.g. TensorFlow), defaults to None
+        :param filters: Custom filters, defaults to None
+        :param search: String to be located in model cards, defaults to ""
+        :return: List of model objects
+        """
+        res = client.get_models(kind=EntryKind.MODEL, task=task, libraries=libraries, filters=filters, search=search)
+        models = []
+
+        for model in res["models"]:
+            res_model = client.get_model(model_id=model["id"])["model"]
+            model_obj = cls(client=client, model_id=model["id"], name=model["name"], description=model["description"])
```

Comment:
This does give warnings for each of the models on creation if their is no schema.